lib/actions/wallet.js

```diff
@@ -173,18 +164,29 @@ function getWalletInfo(walletId, type) {
  * @dispatches
  * @returns {void}
  *
- * NOTE: only supports traditional wallets
  * NOTE: depends on wallets already being in state
  */
 function getWalletsInfo() {
   return async (dispatch, getState) => {
-    const { wallet } = getClient();
+    const { wallet, multisig } = getClient();
     const state = getState();
     const wallets = selectWallets(state);
-    const promises = wallets.map(wid => wallet.getInfo(wid));
-    const results = await Promise.all(promises);
-    for (let i = 0; i < promises.length; i++)
-      dispatch(setWalletInfo(wallets[i], results[i]));
+    const multisigs = selectMultisigWallets(state);
+
+    // fetch info for each standard wallet
+    // then fetch info for each multisig wallet
+    {
+      const promises = wallets.map(wid => wallet.getInfo(wid));
+      const results = await Promise.all(promises);
```

Comment:
I was thinking of a single way of doing errors, it would be really nice if there was an `errorsReducer` built into bpanel that was able to catch errors and dispatch to a place where any plugin could know that there was an error. Then plugin devs don't have to write error handling boilerplate, they can just have data injected from redux that tells them if there was an error

Reply:
Yep, I agree. This has been on the backlog/wishlist for a while, just haven't had the time to put something like this together unfortunately :(.